ops/testing.py

```diff
@@ -1509,17 +1509,12 @@ def _get_backend_calls(self, reset: bool = True) -> List[Tuple[Any, ...]]:
             self._backend._calls.clear()
         return calls
 
-    def add_model_secret(self, owner: AppUnitOrName, content: Dict[str, str]) -> str:
-        """Add a secret owned by the remote application or unit specified.
-
-        This is named :code:`add_model_secret` instead of :code:`add_secret`
-        to avoid confusion with the :meth:`ops.Application.add_secret`
-        and :meth:`ops.Unit.add_secret` methods used by secret owner
-        charms.
+    def add_charm_secret(self, owner: AppUnitOrName, content: Dict[str, str]) -> str:
```

Comment:
I just talked to John Meinel about this. It sounds like they have *not* settled on the name "charm secrets" and there's a bit more discussion they want to have about all this. So I think we should wait with this renaming. We can either just hold off on merging this PR till they've settled that (my recommendation) or we could merge with `add_model_secret` and `add_user_secret` and then change/deprecated `add_model_secret` in a future PR. What do you think @tonyandrewmeyer?

Reply:
Did John have any idea of when they might settle? If the expectation is that it'll be before 2.13 then I would definitely just wait. If it's more likely to be in May (e.g. after Madrid) then I'd leave it to @IronCore864 - if he wants to do the extra work to split this into two PRs I have no objection, but I also don't see any issue with waiting a bit longer. If it might be even further away than that, then I would split them now.

Reply:
No, not really. I got the sense they wanted to decide soon, but I'm not sure how long that piece of string is...

I'd probably recommend not doing the rename now and leaving that for a future PR.

Reply:
(so that we can merge this work without blocking).